StripePaymentSheet/StripePaymentSheet/Source/Helpers/PaymentSheetLinkAccount.swift

```diff
@@ -307,3 +499,96 @@ extension PaymentSheetLinkAccount {
         return params
     }
 }
+
+// MARK: - Payment method availability
+
+extension PaymentSheetLinkAccount {
+
+    /// Returns a set containing the Payment Details types that the user is able to use for confirming the given `intent`.
+    /// - Parameter intent: The Intent that the user is trying to confirm.
+    /// - Returns: A set containing the supported Payment Details types.
+    func supportedPaymentDetailsTypes(for elementsSession: STPElementsSession) -> Set<ConsumerPaymentDetails.DetailsType> {
+        guard let currentSession = currentSession, let fundingSources = elementsSession.linkFundingSources else {
+            return []
+        }
+
+        let fundingSourceDetailsTypes = Set(fundingSources.compactMap { $0.detailsType })
+
+        // Take the intersection of the consumer session types and the merchant-provided Link funding sources
+        var supportedPaymentDetailsTypes = fundingSourceDetailsTypes.intersection(currentSession.supportedPaymentDetailsTypes)
+
+        // Special testmode handling
+        if apiClient.isTestmode && Self.emailSupportsMultipleFundingSourcesOnTestMode(email) {
+            supportedPaymentDetailsTypes.insert(.bankAccount)
+        }
```

Comment:
Do you have any context on why we are automatically adding `.bankAccount` in test mode?

Reply:
This is a special client-side debug account for testing — FC uses it by appending "+multiple_funding_sources@" to the email address.